addons/l10n_es_edi_tbai/data/template_LROE_bizkaia.xml

```diff
@@ -127,7 +127,7 @@
                                 <BaseImponible t-out="format_float(tax['base'])"/>
                                 <TipoImpositivo t-out="tax['rec'].amount"/>
                                 <CuotaIVASoportada t-out="format_float(tax['tax'])"/>
-                                <CuotaIVADeducible t-out="format_float(tax['tax'])"/>
+                                <CuotaIVADeducible t-out="format_float(tax['tax']) if tax['rec'].l10n_es_type != 'no_deducible' else '0.00'"/>
```

Comment:
in the fw-port there are 2 CuotaIVADeducible (one for freelancers and one normal)